packages/kit/src/core/adapt/test/index.js

```diff
@@ -39,9 +39,16 @@ suite('copy files', () => {
 	};
 
 	/** @type {import('types/internal').BuildData} */
-	const build_data = { client: [], server: [], static: [], entries: [] };
+	const build_data = {
+		// @ts-ignore
```

Comment:
we've switched to `@ts-expect-error` if it's something that we want to permanently ignore and are using `@ts-ignore` for temporary todos

Reply:
fixed